src/scss/style.scss

```diff
@@ -1,14 +1,198 @@
+$section-backgroundcolor: rgb(233, 229, 229);
+
 body {
-  background-color: pink;
-  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-  height: 2000px;
+  font-family: serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+section {
+  h2 {
+    font-size: 26px;
+    margin-left: 200px;
+    background: white;
+    padding-left: 6px;
+    border-left: 4px solid #e71212;
+  }
+}
+
+.imageText__content {
+  flex-direction: column;
+  margin: auto auto auto 200px;
+
+  h3 {
+    font-size: 26px;
+  }
+
+  p {
+    font-size: 12px;
+  }
+}
+
+.imageText {
+  background: $section-backgroundcolor;
+
+  &__headline--color {
+    color: #e96262;
+  }
+
+  &__button {
+    color: white;
+    background: black;
+    padding: 8px 12px;
+    margin: auto auto auto 0;
+    border: none;
+    font-size: 10px;
+    font-family: serif;
+  }
+
+  img {
+    width: 50%;
+    max-height: 450px;
+  }
+}
+
+
+//for plain JS
+@keyframes slideDown {
+  from {
```

Comment:
I'm not sure why this ist marked red inside github. Aliases ( from & to ) should work fine.
Alternativly you could use 0% & 100%